core/src/main/java/jenkins/security/UpdateSiteWarningsMonitor.java

```diff
@@ -162,6 +163,11 @@ public boolean hasApplicableHiddenWarnings() {
         return getActiveWarnings().size() < configuration.getApplicableWarnings().size();
     }
 
+    @Override
+    public Permission getRequiredPermission() {
+        return Jenkins.SYSTEM_READ;
```

Comment:
While `doForward` doesn't _do_ anything, it's a small step to something that does, so perhaps just add an unnecessary check?